### PR TITLE
feat(web-app): migrate gatsby-theme video component for mdx usage

### LINKED ISSUE #533: Video MDX component
- [x] add video component
- [x] test inside of do-dont

### PR DESCRIPTION
**Pull request by francinelucca**
Closes #533 

#### Testing / reviewing

http://localhost:3000/mdx/video/video
https://gatsby-theme-carbon.vercel.app/components/Video/

### CODE REVIEW

File: services/web-app/components/video/video.module.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+div.video-button:hover {
+  background-color: colors.$gray-90;
+  opacity: 0.8;
+}
+
+div.video-button > svg {
+  margin-left: 4px;
+  fill: colors.$white-0;
```
Comment (alisonjoseph) [inline services/web-app/components/video/video.module.scss:45]:
```suggestion
  fill: colors.$icon-inverse;
```

File: services/web-app/components/video/video.module.scss [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  outline-offset: -2px;
+}
+
+div.video-button:hover {
+  background-color: colors.$gray-90;
+  opacity: 0.8;
+}
+
+div.video-button > svg {
+  margin-left: 4px;
```
Comment (alisonjoseph) [inline services/web-app/components/video/video.module.scss:44]:
```suggestion
  margin-left: spacing.$spacing-02;
```

**Review by alisonjoseph — COMMENTED** [review]
Couple super small things, but this looks great! 🔥  (not sure what the linting error is)

File: services/web-app/components/video/video.module.scss [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+div.video--is-playing > svg {
+  margin-left: 0;
+}
+
+// iframe
+.vimeo {
+  position: relative;
+  overflow: hidden;
+  height: 0;
+  padding-bottom: 56.543%;
```
Comment (andreancardona) [inline services/web-app/components/video/video.module.scss:59]: this number feels weird (maybe just me..hehe) is there a better way to do this?

**Review by andreancardona — DISMISSED** [review]
Other than my one comment looks good. 🌮 🔥

File: services/web-app/components/video/video.module.scss (hunk 3 above)
Comment (alisonjoseph) [reply services/web-app/components/video/video.module.scss:59]: Oooh, this weirdness is a trick to get the vimeo iframe to be responsive I believe.

Comment (francinelucca) [reply services/web-app/components/video/video.module.scss:60]: yep! looks like a trick to get the video to assume full container width, see: https://stackoverflow.com/questions/42858199/vimeo-full-width
I added a comment in the scss file

**Review by alisonjoseph — DISMISSED** [review]

**Review by andreancardona — DISMISSED** [review]

**Review by alisonjoseph — DISMISSED** [review]

**Review by andreancardona — DISMISSED** [review]

**Review by alisonjoseph — APPROVED** [review]

**Pull request merged**